blocks/test/fixtures/core__button__center.json

```diff
@@ -8,7 +8,8 @@
             "text": [
                 "Help build Gutenberg"
             ],
-            "align": "center"
+			"align": "center",
+			"opensInNewWindow": false
```

Comment:
Indentation issue, I think we're using spaces for the fixtures because they're generated

Reply:
thank you @youknowriad, I did not notice this difference and I corrected to use spaces.